src/gui/src/Action.cpp

```diff
@@ -2,11 +2,11 @@
  * barrier -- mouse and keyboard sharing utility
  * Copyright (C) 2012-2016 Symless Ltd.
  * Copyright (C) 2008 Volker Lanz (vl@fidra.de)
- * 
```

Comment:
Can we split off the whitespace changes to at least a separate commit?

Reply:
My vim automatically remove spaces at the end of line. Sorry for that, but I think it is a good behavior.

Reply:
I agree, just put these changes into a separate commit. Otherwise it will be harder to understand what's happening to whoever to looks into the history later.

Reply:
I reverted all space-at-end-of-line change. There are many the case in this code base.